models/mod.js

```diff
@@ -0,0 +1,15 @@
+var mongoose = require('mongoose');
+var AutoIncrement = require('mongoose-sequence');
+var Schema = mongoose.Schema;
+
+var schema = new Schema({ 
+    name: String, 
+    category: Number, 
+    orbVer: String, 
+    author: String, 
+    pictureLink: String, 
+    filename: String, 
+}, {timestamps: true});
```

Comment:
The current schema has an "External" flag, I don't know if we want to bring that into this one. It's entire purpose is to signify that we don't hold the asset itself for the addon.